index.html

```diff
@@ -1015,45 +1475,47 @@
       <footer class="flex gap-10 flex-col">
         <!-- first div for footer paragraph button background images etc -->
         <div
-          class="relative border-top-bottom grid items-center sm:h-64 overflow-hidden"
+          class="relative border-t border-b border-[#262626] grid items-center sm:h-64 overflow-hidden"
```

Comment:
use shorthand border-y for top and bottm

Reply:
changed